src/main/java/frc/robot/subsystems/ClimbPivot.java

```diff
@@ -34,11 +35,12 @@ public class ClimbPivot extends SubsystemBase {
 
   private final double CLIMB_IN_PRESET = -0.09;
   private final double CLIMB_OUT_PRESET = -0.40;
-  private final double FORWARD_SOFT_STOP = 1;
+  private final double FORWARD_SOFT_STOP = -0.07;
   private final double REVERSE_SOFT_STOP = -78;
   private final double CLIMB_IN_SPEED = 0.2;
   private final double CLIMB_OUT_SPEED = -0.2;
   private final double BOOLEAN_TOLERANCE = 0.05;
+  public final double CLIMB_HOlD = 0.02;
```

Comment:
```suggestion
  private final double CLIMB_HOLD = 0.02;
```
No need to be public, and fix typo.